src/main/resources/static/api.yaml

```diff
@@ -88,4 +141,88 @@ components:
       properties:
         hipId:
           type: string
-          description: Identifier of the health information provider where patient record needs to be looked up.
\ No newline at end of file
+          description: Identifier of the health information provider where patient record needs to be looked up.
+
+    ConsentRequestsRepresentation:
+      type: object
+      properties:
+        limit:
+          type: integer
+        size:
+          type: integer
+        offset:
+          type: integer
+        requests:
+          type: array
+          maxItems: 100
+          items:
+            $ref: '#/components/schemas/ConsentRequestRepresentation'
+
+    ConsentRequestRepresentation:
+      type: object
+      properties:
+        purpose:
+          type: object
+          properties:
+            description:
+              type: string
+        hitypes:
+          type: array
+          items:
+            $ref: '#/components/schemas/HITypeRepresentation'
+        requester:
+          type: object
+          properties:
+            id:
+              type: string
+              format: uuid
+              example: 496413f0-7b63-4f80-9bb0-9382cc3108d3
+            name:
+              type: string
+              example: "Dr. Karuppan"
+            orgName:
+              type: string
+              example: "TMH"
+            contact:
```

Comment:
Contact should be an array preferably. and maybe each "contact" consisting of "type" (mobile, phone, email etc) and value.

Reply:
We will not contact on this request information.